src/components/backend-ai-session-launcher.ts

```diff
@@ -2700,6 +2702,22 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
 
     // monkeypatch for grid items in accessible vfolder list in Safari or Firefox
     this._grid?.clearCache();
+    
+    this._showselectedImg
```

Comment:
To call this function, add `()` to end of sentence (`this._showselectedImg()`).
And I recommend rename to `_showSelectedImg()`.

Reply:
change syntax error and function nm